mmv1/products/dlp/api.yaml

```diff
@@ -309,6 +309,18 @@ objects:
                       - :TOP
                       - :RANDOM_START
                     default_value: :TOP
+                  - !ruby/object:Api::Type::Array
+                    name: 'identifyingFields'
+                    description: |
+                      Specifies the BigQuery fields that will be returned with findings.
+                      If not specified, no identifying fields will be returned for findings.
+                    item_type: !ruby/object:Api::Type::NestedObject
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'name'
+                          required: true
+                          description: |
+                            field name
```

Comment:
I originally thought this was just a placeholder. Might be worth a slightly more useful description, or at least consistent capitalization?

Reply:
Done.